design-at-cornell/src/footer/Footer.tsx

```diff
@@ -35,9 +19,7 @@ const Footer = () => {
           </DTILogo>
         </div>
       </FooterColumnn>
-      {column(explore)}
-      {column(resources)}
-      {column(account)}
+      <p>For any questions, please contact hello@cornelldti.org</p>
```

Comment:
If possible, make the email a link so that users can click on it and it automatically opens an email draft to hello@cornelldti.org.